x-pack/plugin/esql/qa/security/src/javaRestTest/java/org/elasticsearch/xpack/esql/EsqlSecurityIT.java

```diff
@@ -293,19 +314,6 @@ private Response runESQLCommand(String user, String command) throws IOException
             }
             pragmas = settings.build();
         }
-        XContentBuilder query = JsonXContent.contentBuilder();
-        query.startObject();
-        query.field("query", command);
-        if (pragmas != Settings.EMPTY) {
-            query.startObject("pragma");
-            query.value(pragmas);
-            query.endObject();
-        }
-        query.endObject();
-        Request request = new Request("POST", "_query");
-        request.setJsonEntity("{\"query\":\"" + command + "\"}");
```

Comment:
Looks like pragmas were never previously sent in this test. I fixed this for the non-async case as part of the test refactoring.